parser.js

```diff
@@ -85,11 +82,11 @@ function setData(){
 
 	
 	var tempString="";
-	tempString=capFirst(tempData[0]);
+	tempString=tempData[0];

```

Comment:
Create constants for these numbers. It's what's known as a "magic number" because a person with fresh eyes on this code sees a random number accessing an element with no context of what that number does.